cypress/integration/1224-user_administration/363-administrator_opens_the_add_role_page_part_i/add_role_dialog.js

```diff
@@ -0,0 +1,36 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_INST_ADMIN_WITH_AUTHOR } from '../../../support/constants';
+import { USER_ADMINISTRATION_HEADINGS, USER_ADMINISTRAION_BUTTONS } from '../../../support/data_testid_constants';
+
+Given('that the User is logged in as Administrator', () => {
+  cy.login(USER_INST_ADMIN_WITH_AUTHOR);
+});
+And('they are on the User Administration Page', () => {
+  cy.get('[data-testid=menu]').click({ force: true });
+  cy.get('[data-testid=menu-admin-institution-users-button]').click({ force: true });
+});
+When('they click a {string} under a {string}', (button, section) => {

```

Comment:
er det lov å gi disse mer beskrivende navn?
```suggestion
When('they click a {button} under a {section}', (button, section) => {

```

Reply:
Nei, det er en type, ikke en variabel